### PR TITLE
Added launch instance in OpenStack

### PR DESCRIPTION
**Pull request by omkar0496**
Signed-off-by: Omkar Nagarkar <omkarnagarkar0496@gmail.com>

##### TYPE

Chapter No:

Section Title:

##### ADDITIONAL INFORMATION

Points completed:
  
Points not yet completed:

### CODE REVIEW

File: chapter2/launch_instance.md [hunk 1]
```diff
@@ -1,2 +1,32 @@
-- Launch instance in OpenStack: Steps
-- Errors while launching instances
+## Launch Instance in OpenStack
+Instances are nothing but virtual machines inside the cloud. We can launch instance using images, volumes or by using snapshots of previous instance.
+
+After login, Goto ** Project &rarr; Compute &rarr; Images** where all the uploaded images are present. User can create several instances from single image.
```
Comment (KhushbuB) [inline chapter2/launch_instance.md]: This is not bold "Project &rarr; Compute &rarr; Images". Check it.

File: chapter2/launch_instance.md [hunk 2]
```diff
@@ -1,2 +1,32 @@
...
+
+After creating specified image we need to create specified volume, Goto **Project &rarr; Compute &rarr; Volumes &rarr; Create Volume**. To enable persistent storage we need **volume** which is block storage device attached to instance.
+
+
+#### Launch Instance
```
Comment (KhushbuB) [inline chapter2/launch_instance.md]: Use H3 tag here i.e "###"

**Review by KhushbuB — COMMENTED** [review]
Go through these changes.

File: chapter2/launch_instance.md [hunk 3]
```diff
@@ -1,2 +1,32 @@
...
+
+![Flavor](../images/chapter2/OpenStack_Flavor.png "OpenStack_Flavor")
+
+**Networks** provide communication channels to the instance in cloud.
+
+![Networks](../images/chapter2/OpenStack_Networks.png "OpenStack_Networks")
+
+There are some additional options such as **Network Ports** for extra communication channels, **Security Groups** and **Server Groups** for launching instance in specified group, **Key Pair** to allow SSH for newly created instance, **Configuration**, **Scheduler Hints** and **Metadata**.
+
+Then click on Launch Instance button and we can see **Active** instance in system.
```
Comment (son-vyas) [inline chapter2/launch_instance.md]: @omkar0496 , you can make bold "Launch Instance" and unbold the **Active**

Comment (son-vyas) [reply chapter2/launch_instance.md]: Other things are fine

**Comment by omkar0496** [comment]
This PR is replaced with #48

**Pull request closed without merging**